Modules/DevTools/TestFramework/TestRunner/src/CommandLineTestTool.Page.al

```diff
@@ -355,24 +355,25 @@ page 130455 "Command Line Test Tool"
 
         CodeunitTestMethodLine.SetRange("Test Suite", GlobalALTestSuite.Name);
         CodeunitTestMethodLine.SetRange("Line Type", CodeunitTestMethodLine."Line Type"::Codeunit);
-        CodeunitTestMethodLine.SetRange(Name, CodeunitName);
-        if not CodeunitTestMethodLine.FindFirst() then
+        CodeunitTestMethodLine.SetFilter(Name, CodeunitName);
```

Comment:
We need to escape it if we are placing a filter SetFilter(Name,'%1',CodeunitName), otherwise special characters may get in the way. Setfilter is tricky, it could lead to some issues with special characters.

Reply:
I have also thought about this. the problem is that imho there is no optimal solution.
escaping problems have to be solved by the dev who calls the function. unfortunately we can't know if the given special character is a control character for filtering or a character in the object name.

for example at `codeunit 137062 "SCM Sales & Receivables"`. 
How is the application supposed to know whether codeunit "SCM Sales & Receivables" is meant or the two code units "SCM Sales" and "Receivables"? that's why i think the developer who uses the procedure/field has to decide and escape if necessary.

So i solved it like the existing solution (7 lines below) where same problem exist.

Reply:
How do we disable "SCM Sales & Recievables" then or a single method that has & characters in it?
I'm afraid that we may disable too much with this function.

Reply:
Dev should use filters as [documented](https://docs.microsoft.com/en-us/dynamics365/business-central/ui-enter-criteria-filters#symbols).

> If you want to filter on any of these symbols, place the filter expression in single quotes ('<expression with symbol>'). For example, if you wanted to filter on records that start with the text J & V, the filter expression would be 'J & V*'.

Reply:
Can we get a new field / method to disable the tests with filters? We could alternatively implement the method to Get the tests with filter and then specify the same payload to disable the tests? 

I'm worried that the special characters will break the regular usage. The developers often put the special characters in method or codeunit names, and this will be a hard break to all users.

Reply:
Can we do a fix like this:
CodeunitTestMethodLine.SetFilter(Name, CodeunitName);
if CodeunitTestMethodLine.IsEmpty() then
     CodeunitTestMethodLine.SetRange(Name, CodeunitName);
     
 This way if we have a broken filter, it will use the setrange that will escape the values

Reply:
this is a good compromise.
I think the change comes closest to what the developer expects.
I will update the pr.